docs/assets/examples/textgrid/v2/main.go

```diff
@@ -53,33 +53,37 @@ func GetMaroto() core.Maroto {
 	)
 
 	m.AddRow(40,
-		text.NewCol(2, "Left-aligned text"),
-		text.NewCol(4, "Centered text", props.Text{Align: align.Center}),
-		text.NewCol(6, "Right-aligned text", props.Text{Align: align.Right}),
+		text.NewCol(3, "Left-aligned text"),
+		text.NewCol(3, "Centered text", props.Text{Align: align.Center}),
+		text.NewCol(3, "Right-aligned text", props.Text{Align: align.Right}),
+		text.NewCol(3, "Justify-aligned text", props.Text{Align: align.Justify}),
```

Comment:
Could you change back to cols as 2, 4 and 6 and add another row to test this Justify? It's important to have different col sizes to test that the behavior of grid is correct.